pkg/controller/controlplane/valuesprovider.go

```diff
@@ -159,13 +170,14 @@ var configChart = &chart.Chart{
 	Objects: []*chart.Object{
 		// this config is mounted by the shoot-kube-apiserver at startup and should therefore be deployed before the controlplane
 		{Type: &corev1.ConfigMap{}, Name: "authn-webhook-config"},
+		{Type: &corev1.ConfigMap{}, Name: "auditlog-webhook-config"},
 	},
 }
 
 var controlPlaneChart = &chart.Chart{
 	Name:   "control-plane",
 	Path:   filepath.Join(metal.InternalChartsPath, "control-plane"),
-	Images: []string{metal.CCMImageName, metal.AuthNWebhookImageName, metal.AccountingExporterImageName, metal.GroupRolebindingControllerImageName, metal.LimitValidatingWebhookImageName},
+	Images: []string{metal.CCMImageName, metal.AuthNWebhookImageName, metal.AuditlogWebhookImageName, metal.AccountingExporterImageName, metal.GroupRolebindingControllerImageName, metal.LimitValidatingWebhookImageName},
```

Comment:
Please also add a default image to the image vector at `charts/images.yaml`.

Reply:
I was trying to follow authn-webhook in adding this webhook, which does not define an image in charts/images.yaml.
I guess I should make a release for https://github.com/metal-stack/kubernetes-splunk-audit-webhook? Do we have conventions about versioning?

Reply:
When you push a tag on the repo starting with "v" a release will be built. We use standard semantic versioning typical for go... just start with v0.1.0?